01-introduction.Rmd

```diff
@@ -113,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
 This approach is advantageous because it encourages collaboration and avoids 'reinventing the wheel'.
 The package **sf** (covered in Chapter \@ref(spatial-class)), for example, builds on its predecessor **sp**.
 
-Shifts in R's spatial ecosystem have been influenced by shifts in the wider R community, as exemplified by the visualization and data processing packages **ggplot2** and **dplyr** (released in [2007](https://cran.r-project.org/src/contrib/Archive/ggplot2/) and [2014](https://cran.r-project.org/src/contrib/Archive/dplyr/) respectively).
```

Comment:
Yes I know but I think that the tidyverse stuff has certainly led to changes. If there's a better package or set of packages to represent recent changes in R I'm happy to replace this, or tone it down a little. Agree we shouldn't be tied to the tidyverse but should acknowledge that, especially for R newcomers, it is immensly popular.